ci-operator/step-registry/rosa-boundary/nightly-provision/rosa-boundary-nightly-provision-commands.sh

```
@@ -0,0 +1,74 @@
+unzip -q /tmp/awscliv2.zip -d /tmp/
+/tmp/aws/install --install-dir /tmp/aws-cli --bin-dir /tmp/bin
+export PATH="/tmp/bin:${PATH}"
+
+# AWS credentials come from mounted secret via AWS_CONFIG_FILE env var
+# Extract credentials from AWS config file for Terraform
+if [[ -n "${AWS_CONFIG_FILE:-}" ]] && [[ -r "${AWS_CONFIG_FILE}" ]]; then
+  echo "Extracting AWS credentials from ${AWS_CONFIG_FILE}"
+
+  # Read credentials from the config file
+  # Format: [default]\naws_access_key_id = ...\naws_secret_access_key = ...
+  export AWS_ACCESS_KEY_ID=$(grep -A10 "\[default\]" "${AWS_CONFIG_FILE}" | grep "aws_access_key_id" | cut -d'=' -f2 | tr -d ' ')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)